### PR TITLE
Added build script for Elastic

### PR DESCRIPTION
**Pull request by atharv-phadnis**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?

Package Name: **elastic**
Package Version: **v7.0.22**
Package URL: [elastic](https://github.com/olivere/elastic/)

Tested on: **UBI 8.5**
Build Errors: **None**
Test Errors: **None**

### CODE REVIEW

File: e/elastic/elastic_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: github.com/olivere/elastic/v7
+# Version	: v7.0.22
+# Source repo	: https://github.com/olivere/elastic
+# Tested on	: UBI 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
```
Comment (seth-priya) [inline e/elastic/elastic_ubi_8.5.sh:9]: Looks like Travis-Check should be mentioned and False for this one? The "Language" field is also missing

Comment (atharv-phadnis) [reply e/elastic/elastic_ubi_8.5.sh:9]: Travis-Check is confusing in this case. I've tested the script on a container, but with the docker socket exposed to it. Had discussed with Ryan as well, even he was unsure about these cases. Maybe you could confirm if I should set it to True or False considering the scenario.

Will add the Language field along with Travis-Check.

Please give your suggestion...

Comment (seth-priya) [reply e/elastic/elastic_ubi_8.5.sh:9]: I don't think Travis-Check will work for this, let's keep it as false @atharv-phadnis

Comment (atharv-phadnis) [reply e/elastic/elastic_ubi_8.5.sh:11]: Updated, please review

**Pull request merged**